app/views/handlers/individual_response.py

```diff
@@ -139,7 +138,17 @@ def handle_get(self):
             self.individual_section_id
         )
 
-        if self._list_item_id:
+        if self._request_args.get("journey") == "remove-person":
+            previous_location_url = url_for(
+                "questionnaire.block",
+                list_name=self._list_name,
+                list_item_id=self._list_item_id,
+                block_id=self._schema.get_remove_block_id_for_list(
+                    self._schema.get_individual_response_list()
```

Comment:
You already have the list name `self._list_name`, so don't need to fetch it again.